### PR TITLE
Use global CSS vars

### PR DESCRIPTION
**Pull request by julio8a**
- switch
- loader
- tree

### Switch
- I'm not sure about the changes to the switch, seems odd, but it works out at the end as the correct values render on the page. Open to feedback or changes if needed. 

### Loader
- @paulcpederson, I noticed `spot-light` and `spot-dark` were using `v-xx-xxx` values. I just changed them to the main blue for light and dark theme. Not sure if we still need both regular `loader-spot` and `loader-spot-light`.

### CODE REVIEW

File: src/components/calcite-loader/calcite-loader.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 :host {
-  --calcite-loader-spot: #{$ui-blue};
-  --calcite-loader-spot-light: #{$v-bb-160};
-  --calcite-loader-spot-dark: #{$h-bb-070};
+  --calcite-loader-spot: var(--calcite-ui-blue);
```
Comment (macandcheese) [inline src/components/calcite-loader/calcite-loader.scss]: I think these need to pass the previous values - if you look at the demo page this seems to break the animation

File: src/components/calcite-switch/calcite-switch.scss [hunk 2]
```diff
@@ -24,36 +24,36 @@
   --calcite-switch-track-border: #{$blk-160};
   --calcite-switch-handle-background: #{$blk-200};
   --calcite-switch-handle-border: #{$blk-100};
-  --calcite-switch-hover-handle-border: #{$ui-blue-hover-dark};
+  --calcite-switch-hover-handle-border: var(--calcite-ui-blue-hover-dark);
```
Comment (macandcheese) [inline src/components/calcite-switch/calcite-switch.scss]: This is what I'm seeing for dark theme - no color for red / blue is being applied for hover or active: 
Screen Shot 2020-01-21 at 4 28 30 PM

File: src/components/calcite-loader/calcite-loader.scss (hunk 1 above)
Comment (julio8a) [reply src/components/calcite-loader/calcite-loader.scss]: fixed, take another look.

File: src/components/calcite-switch/calcite-switch.scss (hunk 2 above)
Comment (julio8a) [reply src/components/calcite-switch/calcite-switch.scss]: I fixed this one too, take another look.

**Comment by macandcheese** [comment]
Looks good now, thanks!

**Pull request merged**